api_helpers.py

```diff
@@ -284,12 +286,15 @@ def create_hs_res_from_czo_row(czo_res_dict, czo_hs_account_obj, index=-99, ):
         # parse czo_id
         czo_id = _extract_value_from_df_row_dict(czo_res_dict, "czo_id")
         record_dict["czo_id"] = czo_id
-        logging.info("Working on Row {index} CZO_ID {czo_id}".format(index=index, czo_id=czo_id))
+        logging.info("Working on NO.{index} CZO_ID {czo_id}".format(index=index, czo_id=czo_id))
 
         # parse CZOS
         czos = _extract_value_from_df_row_dict(czo_res_dict, "CZOS")
         czos_list = czos.split('|')
         czo_primary = czos_list[0]
```

Comment:
You can reduce lines of code with chaining
czos.split('|')[0]

consider error handling for index out of bounds if it yielded none... then that also integrates with a simplified conditional (if) in line 295... hack around see what you can come up with